geode-core/src/main/java/org/apache/geode/internal/cache/DiskDirectoryStats.java

```diff
@@ -99,6 +99,10 @@ public long getDiskSpace() {
     return this.stats.getLong(diskSpaceId);
   }
 
+  public long getMaxSpace() {
+    return this.stats.getLong(maxSpaceId);
+  }
+
```

Comment:
I don't see where this is used. I recommend removing it if it's not needed.

Reply:
Agree, this is from a previous version of my code, not needed here.